src/sentry/receivers/groups.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import print_function, absolute_import
+
+from sentry import analytics
+from sentry.signals import group_created
+
+
+@group_created.connect(weak=False)
+def record_join_request_created(group, project, organization, **kwargs):
```

Comment:
diff func name maybe?

Reply:
thanks, forgot to rename this from the example i was looking at